external.config.js

```diff
@@ -1,7 +1,12 @@
 var path = require('path');
-var webpack = require('webpack');
+var UglifyJsPlugin = require('uglifyjs-webpack-plugin');
 
 module.exports = {
+  /* webpack 4
+  mode: 'production',
+   */
+  performance: {hints: false},
```

Comment:
what does this do?

Reply:
It suppresses certain warning messages.  Webpack, for instance, complains that the proj4 inclusion is large and could be broken into smaller modules, but we want proj4 as it stands, so this is not a useful warning.  Rather than see this on every build, this turns those warnings off.

Reply:
thanks